2-advanced/dubbo-samples-seata/case-configuration.yml

```diff
@@ -80,76 +67,61 @@ services:
     depends_on:
       - seata-mysql
       - seata-server
-      - nacos
 
-  account-service:
+  order-service:
     type: app
-    basedir: dubbo-samples-seata-account
-    mainClass: org.apache.dubbo.samples.seata.AccountApplication
+    basedir: dubbo-samples-seata-order
+    mainClass: org.apache.dubbo.samples.seata.account.OrderApplication
     systemProps:
-      - nacos.address=nacos
-      - nacos.port=8848
       - seata.address=seata-server
     jvmFlags:
       - "--add-opens java.base/java.lang.reflect=ALL-UNNAMED"
     waitPortsBeforeRun:
-      - nacos:8848
       - seata-mysql:3306
       - seata-server:8091
     checkLog: "Current Spring Boot Application is await"
     checkPorts:
       - 20881
     depends_on:
-      - nacos
       - seata-mysql
       - seata-server
 
-  order-service:
+  stock-service:
     type: app
-    basedir: dubbo-samples-seata-order
-    mainClass: org.apache.dubbo.samples.seata.OrderApplication
+    basedir: dubbo-samples-seata-stock
+    mainClass: org.apache.dubbo.samples.seata.account.StockApplication
     systemProps:
-      - nacos.address=nacos
-      - nacos.port=8848
       - seata.address=seata-server
     jvmFlags:
       - "--add-opens java.base/java.lang.reflect=ALL-UNNAMED"
     waitPortsBeforeRun:
-      - nacos:8848
       - seata-mysql:3306
       - seata-server:8091
     checkLog: "Current Spring Boot Application is await"
     checkPorts:
       - 20883
     depends_on:
-      - nacos
       - seata-mysql
       - seata-server
-      - account-service
 
   business-test:
     type: test
     basedir: dubbo-samples-seata-business
     tests:
-      - "**/*IT.class"
+      - "**/*Tests.class"
```

Comment:
In dubbo-sample, *IT.class is a unified suffix, which can facilitate narrative, pls restore this.

Reply:
OK

Reply:
Done